src/internal/compilerSupport/v3impl/misc.ts

```diff
@@ -0,0 +1,160 @@
+import { SecurityTetraplet } from '@fluencelabs/avm-runner-interface';
```

Comment:
why `misc.ts`? there is all about services except call params extraction. Maybe move exctraction function and rename?